test/Faker/Provider/ar_EG/CompanyTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Faker\Test\Provider\ar_EG;
+
+use Faker\Calculator\Luhn;
+use Faker\Provider\ar_EG\Company;
+use Faker\Test\TestCase;
+
+/**
+ * @group legacy
+ */
+final class CompanyTest extends TestCase
+{
+    public function testCompanyPrefix()
+    {
+        $companyPrefix = $this->faker->companyPrefix();
+        self::assertIsString($companyPrefix);
```

Comment:
Please test for a specific string as output, our test's Faker instance is seeded with `1`, it will always provide the same output.

Reply:
If you could nudge me in the right direciton that would be appreciated.
i could test that the output matches a regular expression as used in \Test\Provider\Company to test a job title.
would that suffice in all places where i used assert string?

Reply:
You need to test what the output of companyPrefix() is when faker is seeded with 0

Reply:
i believe latest commit covers all the issues